package.json

```
@@ -41,7 +41,7 @@
     "typescript": "5.0.4"
   },
   "dependencies": {
-    "@synonymdev/slashtags-url": "1.0.0-alpha.3",
+    "@synonymdev/slashtags-url": "^1.0.0-alpha.4",
```

```suggestion
    "@synonymdev/slashtags-url": "1.0.0-alpha.4",
```

Hey @Nuhvi for that dependency, mind using `npm install --exact`? 
Let's keep it at `1.0.0-alpha.4` to avoid any alpha surprises.

---

Updated to non-alpha version.